File: python-collections/dictionary/word_count.py
#word="hello hai hello hai hello"
#count no:of words
#hello:3,hai:2

text="hai hello hai hello"
words=text.split(" ")
#words=[hai,hello,hai,hello]

dict={}
#created an empty dictionary
#key value
#hai 1
#hello 1


for word in words:#word=hai,hello
    if(word not in dict):#if hai not in dictionary hai:1,hello:1
        dict[word]=1
    else:
        dict[word]+=1#dict["hello'] 1+1=2
print(dict)

File: Html_sections/htmlsections/luproject/luminar-project/Javascript/objects_programs/wordcount.js
var text="hai hello hai hello";

dict={};
var words=text.split(" ");
//console.log(words);

for(let word of words){
    if(word in dict){
        dict[word]+=1;
    }
    else{
        dict[word]=1;
    }


}
console.log(dict);